extensions/amp-fx-collection/amp-fx-collection.md

```diff
@@ -79,6 +79,38 @@ In this example, as the user scrolls the page, the h1 element scrolls faster rel
 </h1>
 ```
 
+### fade-in
```

Comment:
New compared to #14276

Reply:
Awesome, thanks for updating the docs in the same PR!

I would add a note about fade-in being experimental.